### ISSUE TITLE
Emojis are being sent okay, but are not shown correctly in the keyboard

### ISSUE THREAD

**Issue by carlmo** [nikhiljha/pp-fedora-sdsetup#127]
I think the pictures will make it clearer, so here it is both from my android phone (signal app) and the pinephone chats app.

I am not sure how to troubleshoot this to be honest. The good news is that it sends it fine, so I guess maybe there's an easy fix ?

![signal-2022-10-13-233939_002](https://user-images.githubusercontent.com/14909968/195757751-364176f9-dc90-47d1-8e1c-a4fd04348ecb.jpeg)
![signal-2022-10-13-234038_002](https://user-images.githubusercontent.com/14909968/195757858-2bed040f-72bc-4fdd-a047-c0f4e6342595.jpeg)
![signal-2022-10-13-234031_002](https://user-images.githubusercontent.com/14909968/195757865-e955a56d-7c6b-422d-b683-0661fcb99681.jpeg)
![signal-2022-10-13-234026_002](https://user-images.githubusercontent.com/14909968/195757872-f3661299-8980-4c89-a885-f03aad06e2cf.jpeg)

**Comment by carlmo** [comment]
Some emojis are fine and visible, but most of them are not okay as you can see

**Comment by Torbuntu** [comment]
You'll need to install an emoji font package. Something like google-noto-emoji-color-fonts

**Comment by carlmo** [comment]
Yep, dnf install google-noto-emoji-color-fonts fixed it. Can we include it in the base image ? 

I'm not a fan of emojis myself, but it seems like if we're aiming at having a phone functionality, we should include this.

**Comment by Torbuntu** [comment]
Makes sense to add I suppose. I think also there are quite a few more packages from the workstation image that would be nice to have. But for testing and such using minimal without a lot of extra packages makes things a little quicker to build and test.
